claat/proto-renderer/generic_renderer_test.go

```diff
@@ -0,0 +1,59 @@
+package genrenderer
+
+import (
+	"go/build"
+	"path/filepath"
+	"testing"
+	"text/template"
+)
+
+type encapsulatedTest struct {
+	In      sampleProtoTemplate
+	Out     string
+	WantErr bool
```

Comment:
any clearer naming? `ShouldPanic`/`ShouldErr`?

Reply:
Conventionally these should be "in", "out" and "ok". Non-public and without want/should.

Reply:
so copying your fetching PR's format? sg